### PR TITLE
Bugfix/login failure due email capitalisation

### PR DESCRIPTION
**Pull request by voyager163**
## What does this PR do?

This code will prevent the email check from failing if the user email contain capitalisation. The current code will result in the user failing the email checks by showing "No Account was found" if the user email contain capitalisation.

## Type of change

- [x]  Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [x] Application can be tested by using capitalisation email address.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code and corrected any misspellings
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by agustif** [comment]
Sweet and simple, and improves UX, looks goos to me. i think we could safely remove the warning from the README

**Comment by voyager163** [comment]
> Sweet and simple, and improves UX, looks goos to me. i think we could safely remove the warning from the README

The issue is that user may still insert capitalisation letters when they insert the user.

**Comment by agustif** [comment]
> > Sweet and simple, and improves UX, looks goos to me. i think we could safely remove the warning from the README
> 
> 
> 
> The issue is that user may still insert capitalisation letters when they insert the user and that is how i found this issue.
> 
> 

Maybe we need to apply a sinilar lowercasing in our private signup will look into that what do yo think @PeerRich

**Comment by PeerRich** [comment]
> > > Sweet and simple, and improves UX, looks goos to me. i think we could safely remove the warning from the README
> > 
> > 
> > 
> > The issue is that user may still insert capitalisation letters when they insert the user and that is how i found this issue.
> > 
> > 
> 
> Maybe we need to apply a similar lowercasing in our private signup will look into that what do yo think @PeerRich 

likely. wanna check?

**Comment by voyager163** [comment]
> > > > Sweet and simple, and improves UX, looks goos to me. i think we could safely remove the warning from the README
> > > 
> > > 
> > > The issue is that user may still insert capitalisation letters when they insert the user and that is how i found this issue.
> > 
> > 
> > Maybe we need to apply a similar lowercasing in our private signup will look into that what do yo think @PeerRich
> 
> likely. wanna check?

Haha, i wanted to help to check but i do not think i have access to the codes. Let me know if you need me to work on it.

**Comment by agustif** [comment]
> > > > > Sweet and simple, and improves UX, looks goos to me. i think we could safely remove the warning from the README
> 
> > > > 
> 
> > > > 
> 
> > > > The issue is that user may still insert capitalisation letters when they insert the user and that is how i found this issue.
> 
> > > 
> 
> > > 
> 
> > > Maybe we need to apply a similar lowercasing in our private signup will look into that what do yo think @PeerRich
> 
> > 
> 
> > likely. wanna check?
> 
> 
> 
> Haha, i wanted to help to check but i do not think i have access to the codes. Let me know if you need me to work on it.
Thank you!
I am already working on improving the signup. 

I will make sure to make a fix for this so registration will cover lowercasing soon 🙏

**Comment by voyager163** [comment]
> Thank you! I am already working on improving the signup.
> 
> I will make sure to make a fix for this so registration will cover lowercasing soon 🙏

No worries man, after this, i will see the rest of the bugs, to see how to close it for you all.

**Review by pumfleet — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -184,6 +184,9 @@ yarn dx
 1. Click on the `User` model to add a new user record.
 1. Fill out the fields `email`, `username`, `password`, and set `metadata` to empty `{}` (remembering to encrypt your password with [BCrypt](https://bcrypt-generator.com/)) and click `Save 1 Record` to create your first user.
    > New users are set on a `TRIAL` plan by default. You might want to adjust this behavior to your needs in the `apps/web/prisma/schema.prisma` file.
+   
+   > :warning: **Please do not use capitalisation in your email**
```
[the remark this answers is not in the document]
Comment (voyager163) [reply README.md:195]: Okie. i will remove it.

**Review by pumfleet — APPROVED** [review]

**Pull request merged**